AnswerContainer.cpp

```diff
@@ -1,22 +1,21 @@
-#include "answercontainer.h"

```

Comment:
We need to put in git config identical setting core.autocrlf to not overwrite changes each time.